common/app/controllers/EmailSignupController.scala

```diff
@@ -58,7 +60,15 @@ class EmailFormService(wsClient: WSClient, emailEmbedAgent: NewsletterSignupAgen
 
   def submit(form: EmailForm)(implicit request: Request[AnyContent]): Future[WSResponse] = {
     val consentMailerUrl = serviceUrl(form, emailEmbedAgent)
-    val consentMailerPayload = JsObject(Json.obj("email" -> form.email, "set-lists" -> List(form.listName)).fields)
+    val consentMailerPayload = JsObject(
+      Json
+        .obj(
+          "email" -> form.email,
+          "set-lists" -> List(form.listName),
+          "set-consents" -> form.marketing.map(_ => List("similar_guardian_products")),
```

Comment:
Why is marketing an `Option[String]` instead of something like an `Option[Boolean]`?

Slight concern that the API could easily be misunderstood and downstream we might accidentally set something like `marketing=false` expecting it not to opt users in.

Reply:
I followed the default html specification for checkboxed where if a string `value` isn't provided, the default value is a string `on`

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#value

Reply:
So thought it would be better to base it on the existence of the key, rather than any specific value. We could change this if WebX decide to implement this in a different way.

Reply:
Ah interesting, thats not how I would have expected checkboxes to work!